go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/dchest/siphash v1.2.1
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/djherbis/atime v1.0.0
+	github.com/dswarbrick/smart v0.0.0-20190505152634-909a45200d6d
```

Comment:
fyi there is a more advanced and up-to-date library to deal with SMART information from golang

https://github.com/anatol/smart.go

Reply:
Yes we are planning to migrate @anatol - if you have some free cycles feel free to send us a PR.